CHANGELOG.md

```diff
@@ -26,6 +26,16 @@
   Instead of it aggregate `near_peer_message_received_by_type_total` metric.
   For example, to get total rate of received messages use
   `sum(rate(near_peer_message_received_by_type_total{...}[5m]))`.
+* Few changes to `view_state` JSON RPC query:
+  - The requset has now an optional `include_proof` argument.  When set to
+    `true`, response’s `proof` will be populated.
+  - The `proof` within each value in `values` list of a `view_state` response is
+    now deprecated and will be removed in the future.  Client code should ignore
+    the field.
+  - The `proof` field directly within `view_state` response is currently always
+    sent even if proof has not been requested.  In the future the field will be
+    skipped in those cases.  Clients should accept responses with this field
+    missing (unless they set `include_proof`).
```

Comment:
Can we just slap `skip(if_empty)` on the `proof` then?

Reply:
I’m worried about clients who expect `proof` field and fail parsing if it’s missing.  Because of that I want to keep sending the empty field for a couple of releases and this release note is a deprecation warning to let users know we’re going to stop sending the empty fields.  This is also mentioned in https://docs.near.org/api/rpc/contracts#view-contract-state (expand the Example response).

Reply:
Ah, I didn't realise that this field was there for a long time, thought we've just added that.